HHS/Head-Start-TTADP
# Update backend dependencies

## Description of change

Update backend dependencies that were relatively easy to update.

Only small changes were required to get the code to work with new packages. Most of the changes were around required eslint changes that honestly might not even be required if I can't get eslint updated. Other big change was around how models are loaded in `src/models/index.js` since sequelize removed the `sequelieze.import` method.

Another positive is I was able to completely remove the audit ignored issues file. `yarn audit` only shows three moderate vulnerabilities, one for jest and the other two for our API documentation viewer.

There are a few packages that I haven't updated (yet!):

`@cucumber` - Don't want to update to release candidate
`csv-*` - Had a hard time with these, might try again
`eslint` - The eslint plugin we use to use multiple formatters blows up when eslint is updated. Will explore solutions
`jest*` - Also had a hard time updating these
`file-type` - Caused some errors when I updated

I will take another pass updating these packages, specifically `file-type`, `eslint` and the `csv-*` packages. Jest feels like a larger project and cucumber isn't _really_ out of date.

`yarn outdated` before:

![Screenshot from 2022-01-06 15-46-09](https://user-images.githubusercontent.com/6631790/148457662-4574964f-9d94-4418-8472-9fa62ca12679.png)

and after:

![Screenshot from 2022-01-07 13-47-03](https://user-images.githubusercontent.com/6631790/148598817-eacbf7b1-b7c0-4f81-b0da-58e7801121d8.png)

## How to test

Browse the site in sandbox and make sure everything still works. Run migrations and other scripts locally to verify they still function properly.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-458

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
